### PR TITLE
Coding: Events top section added

### LINKED ISSUE #90: Coding: Develop the following Events Section
## Description
As a developer,
I need the following section to be developed,
so that I can add it as an Events component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
Develop the section in the red box only
event1

Font Size for heading EVENTS: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

[Design Link for Development](https://xd.adobe.com/view/237a598d-ce4a-4fee-5642-3048a38e6a8b-554b/)

### Update [Required]
- [ ] Should follow all the margin guidelines mentioned above and should be responsive

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6-8 hours

### PR DESCRIPTION
**Pull request by anna4j**
### Description

Created the top component of the **Events** section along with fetching data from respective json content.
Implemented the following feature for responsiveness
platform
So the image and text wrap row-wise in web and column-wise in mobile screens

The web view is as shown: 
view

Fixes #90 

### Type of Change:
**Delete irrelevant options.**

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/Components/Events/Event.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import { Dimensions } from "react-native";
+
+
+export default class Event extends Component {
+
+    render() {
+        return (
+            <View style={[styles.container]}>
+                <View style={[styles.left]}>
+                    <Text style={[styles.heading]}>{content['sections'][0].title}</Text>
```
Comment (annabauza) [inline src/Components/Events/Event.js:16]: Please reuse SectionHeder for consistency.

File: src/Components/Events/Event.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    <Text style={[styles.description]}>
+                        {content['sections'][0].content}</Text>
+                </View>
+                <View style={[styles.middle]} />
+                <View
+                    style={{
```
Comment (annabauza) [inline src/Components/Events/Event.js:22]: move styles outside of component and attach reference.

File: src/Components/Events/Event.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+const styles = StyleSheet.create({
+    container: {
+        backgroundColor: 'white',
+        width: '100%',
+        flexWrap: 'wrap',
+        flexDirection: (Platform.OS === 'web') ? 'row' : 'column',
+    },
+
+    heading: {
+        fontSize: '2.55em',
```
Comment (annabauza) [inline src/Components/Events/Event.js:49]: we don't use em, this will break mobile native

File: src/Components/Events/Event.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        fontWeight: 500,
+        paddingBottom: 5,
+        paddingTop: 40,
+        color: '#103B81',
+    },
+
+    description: {
+        fontSize: '1.4em',
```
Comment (annabauza) [inline src/Components/Events/Event.js:57]: we don't use em, this will break mobile native

File: src/Components/Events/Event.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        paddingBottom: 32,
+
+    },
+
+    left: {
+        width: (Platform.OS === 'web') ? '50%' : '100%',
+    },
+
+    middle: {
+        width: '2.3em',
```
Comment (annabauza) [inline src/Components/Events/Event.js:69]: we don't use em, this will break mobile native

File: src/Components/Events/Event.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    },
+
+    img: {
+        width: (Platform.OS === 'web') ? '43.4em' : '24.5em',
```
Comment (annabauza) [inline src/Components/Events/Event.js:73]: we don't use em, this will break mobile native

File: src/Components/Events/Event.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+        height: (Platform.OS === 'web') ? '12em' : '6.5em',
```
Comment (annabauza) [inline src/Components/Events/Event.js:74]: we don't use em, this will break mobile native

File: src/Components/Events/index.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+    return (
+        <View style={{
+
+            paddingLeft: 32,
+            paddingRight: 32,
+        }}>
+            <Event />
+            <View style={{
+                borderWidth: '0.05em',
```
Comment (annabauza) [inline src/Components/Events/index.js:16]: we don't use em, this will break mobile native

File: src/Components/Events/index.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { View } from 'react-native';
+import Event from './Event';
+
+
+const Events = () => {
+
+    return (
+        <View style={{
```
Comment (annabauza) [inline src/Components/Events/index.js:9]: move style from component and add reference

File: src/Components/Events/index.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+            paddingLeft: 32,
+            paddingRight: 32,
+        }}>
+            <Event />
+            <View style={{
```
Comment (annabauza) [inline src/Components/Events/index.js:15]: move style from component and add reference

**Comment by anna4j** [comment]
@Sonal240 hey! I need some time to make changes to this code. I'm still working on a few changes, will update you if I get stuck somewhere. Thanks!

**Comment by nandini45** [comment]
@anna4j  can u tell us till when can you update your work?
you have two pending PRs.

**Comment by anna4j** [comment]
@nandini45 Hey, I'll do both PR updates within this week positively. Really sorry about the delay!

**Comment by saideepesh000** [comment]
> 
> 
> @nandini45 hey, just need some more time. Hoping to open PR soon. Sorry for the delay :(

@anna4j, Hey use this package [react-native-reflect](https://sntx.github.io/react-native-reflect/#/?id=single-style) for responsiveness.

**Comment by annabauza** [comment]
@anna4j I'm taking this over since you asked for more time 21 days ago. Thanks for contribution.

**Pull request closed without merging**